mParticle-Leanplum/MPKitLeanplum.m

```diff
@@ -79,8 +79,11 @@ - (void)start {
     
     dispatch_once(&kitPredicate, ^{
         [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(onUserIdentified:) name:mParticleIdentityStateChangeListenerNotification object:nil];
+        NSDictionary *identities = [[self currentUser] userIdentities];
+        NSString *advertiserId = identities[@(MPIdentityIOSAdvertiserId)];
+
         if ([MParticle sharedInstance].environment == MPEnvironmentDevelopment) {
-            LEANPLUM_USE_ADVERTISING_ID;
```

Comment:
reading through the git history - this line seems like a relic that has somehow persisted and is actually now just a bug - I think we can remove it. dev vs prod shouldn't make an impact here.

Reply:
@samdozor So you're saying to not set advertiser ID here?

Reply:
ie just remove don't replace

Reply:
exactly - the logic for setting device id is below, we don't need/want this undocumented special-case whereby we use idfa in development mode.